content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: "From Code to Quill: Embark on a Legendary Kubernetes Quest with SIG-Docs"
+date: 2024-04-28
+slug: contribute-as-sig-docs-reviewer
+author: >
+   Ricardo Amaro (Acquia)
+---
+
+You've probably come across the saying, "Contributing doesn't always mean writing lines of code"
+whispered to the wind or glimpsed in the margins of a sacred open-source tome.
+What is the truth behind this tale as old as time (or at least as old as Kubernetes itself)?
+
+Imagine Sarah: an enthusiast of open-source whose keyboard had seen more action crafting
+technical masterpieces for her projects than a knight's sword in battle. Sarah's had such
+a smooth relationship with Git, that she could `git commit` and `git push` in her sleep,
+and her etiquette in pull-requests was so polished, it could make a silver spoon look tarnished.
+
+But here's the twist: when it came to the mysterious _art-of-coding_, Sarah felt more
+like a fish out of water — or more accurately, like trying to navigate a Kubernetes cluster
+with a compass and a star map. The very thought of diving into the codebase made her feel like
+she was trying to read ancient hieroglyphs without a Rosetta Stone.
+
+Did Sarah let this stop her? Not at all. She discovered a path as impactful as a well-placed
+line of code: joining the Special Interest Group (SIG) Docs team. It was there that Sarah found her true calling,
```

Comment:
nit: SIG Docs is already a group/team per the acronym, so adding the word "team" after is redundant